architecture-decision-records/0006-user-data-model.md

```diff
@@ -0,0 +1,61 @@
+# 6. User data model
+
+**Date**: 2020-03-06
+
+## Context
+
+We have [decided][user-auth-adr] to use the OAuth2 provider Keycloak for user
```

Comment:
Appears to be a broken link to the [user-auth-adr]

Reply:
Fixed, thanks.